crates/nu-cli/src/prompt_update.rs

```diff
@@ -99,12 +99,10 @@ fn get_prompt_string(
 pub(crate) fn update_prompt(
     config: &Config,
     engine_state: &EngineState,
-    stack: &Stack,
+    stack: &mut Stack,
     nu_prompt: &mut NushellPrompt,
 ) {
-    let mut stack = stack.clone();
```

Comment:
This clone is removed, but when we call `update_prompt` we are cloning above, to make sure any changes from `get_prompt_string` are isolated